scripts/global-food-explorer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas>=1.2.0
+
```

Comment:
Did you consider using [poetry](https://python-poetry.org/)?

Reply:
Yes I did, shortly. I disregarded it since it is only really used to set up the GH Action (I would assume that most of us have a Python installation with pandas dangling around somewhere).

But now that you bring this up again I'm gonna revisit poetry and probably switch over sometime later today.